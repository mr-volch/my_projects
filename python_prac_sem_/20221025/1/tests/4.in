print(*enumerate(fib(25,40)))
