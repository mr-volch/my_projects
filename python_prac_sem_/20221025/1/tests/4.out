(0, 121393) (1, 196418) (2, 317811) (3, 514229) (4, 832040) (5, 1346269) (6, 2178309) (7, 3524578) (8, 5702887) (9, 9227465) (10, 14930352) (11, 24157817) (12, 39088169) (13, 63245986) (14, 102334155) (15, 165580141) (16, 267914296) (17, 433494437) (18, 701408733) (19, 1134903170) (20, 1836311903) (21, 2971215073) (22, 4807526976) (23, 7778742049) (24, 12586269025) (25, 20365011074) (26, 32951280099) (27, 53316291173) (28, 86267571272) (29, 139583862445) (30, 225851433717) (31, 365435296162) (32, 591286729879) (33, 956722026041) (34, 1548008755920) (35, 2504730781961) (36, 4052739537881) (37, 6557470319842) (38, 10610209857723) (39, 17167680177565)
